documents/knowledge/elements.yaml

```
@@ -65,17 +65,17 @@
     - "[1, 2] : 1"
 
 - element: "⟇"
-  name: Reversed Logical Or
+  name: Remove at Index
   arity: 2
-  description: Returns  the same as logical or but with reversed arguments.
+  description: Returns every item in a list except the item at the given index.
   overloads:
-    any-any: b or a
+    any-num: Remove item b of a
   vectorise: false
   tests:
-    - "[0, 0] : 0"
-    - '["", 1] : 1'
-    - "[[1,2,3], 0] : [1,2,3]"
-    - "[1, 2] : 2"
+    - "[[1,2,3], 0] : [2,3]"
+    - "[[1,2,3], 1] : [1,3]"
+    - "[[1,2,3,1], 3] : [1,2,3]"
+    - "[[1,2,3,1], 0] : [2,3,1]"
```

```suggestion
    - "[3, [1,2,3,1]] : [1,2,3]"
    - "[0, [1,2,3,1]] : [2,3,1]"
```
imma just swap these around so we can confirm that the second overload works